### ISSUE TITLE
Filters on relationships

### ISSUE THREAD

**Issue by devexsolutions** [thephpleague/fractal#522]
Hi, how can I apply filters on relationships?

**GET /comments?filter[post]=1,2&filter[author]=12 HTTP/1.1**

Thanks

**Comment by alkhachatryan** [comment]
The same question from me.

**Comment by matthewtrask** [comment]
Fractal is a serializer and data transformer, so it doesn't really look at what you are passing to the request. You would need to handle the filter in your own code and then conditionally handle the data being sent to your transformer.

**Comment by willishq** [comment]
I agree, however I think in the documentation there is a reference to the querystring being used to parse includes, which could cause some confusion.

Maybe we need to change the documentation to be explicit that Fractal does not choose what data you pass to it (such as posts with a specific ID or Author) and instead manages on how to display the data passed to it.

Interestingly, as an aside, I did try to solve this problem 3 years ago with a package I called query-grid, however the concept is pretty complex and it's probably easier to write things like filtering by hand per use-case rather than try to create a catch-all package.
